### PR TITLE
feat: changed multiple sentences

### PR DESCRIPTION
**Pull request by svenjens**
Description 
This PR takes care of a couple of text changes. 
I will add tesst if edgar approved the changes. 

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [ ] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [ ] you added the necessary documentation (either in the markdown files or inline)
- [ ] you added the necessary automated tests

### CODE REVIEW

File: apps/client/src/components/Location/LocationInput.js [hunk 1]
```diff
@@ -144,6 +144,7 @@ const LocationInput = ({
           address={sessionAddress}
         />
         <Nav
+          nextText={hasSTTR ? "Naar de vragen" : "Volgende"}
```
Comment (robinpiets) [inline apps/client/src/components/Location/LocationInput.js:148]: 👍

File: apps/client/src/components/RegisterLookupSummary.js [hunk 2]
```diff
@@ -44,19 +44,12 @@ const RegisterLookupSummary = ({
   return (
     <ComponentWrapper marginBottom={sttrFile ? "0" : null}>
       <Paragraph gutterBottom={16}>
-        {compact ? (
+        <>
```
Comment (robinpiets) [inline apps/client/src/components/RegisterLookupSummary.js]: This Fragment is probably not needed

File: apps/client/src/components/Location/ChangeAddressModal.tsx [hunk 3]
```diff
@@ -82,7 +83,10 @@ const ChangeAddressModal: React.FC<{
           showConfirmButton
         >
           <ComponentWrapper>
-            <Paragraph>Lorem ipsum</Paragraph>
+            <Paragraph>
+              Alle eerder gegeven antwoorden worden gewist. Weet u zeker dat u
```
Comment (robinpiets) [inline apps/client/src/components/Location/ChangeAddressModal.tsx]: As discussed...

File: apps/client/src/components/Location/ChangeAddressModal.tsx [hunk 4]
```diff
@@ -61,10 +61,11 @@ const ChangeAddressModal: React.FC<{
     <>
       {hasSTTR ? (
         <Modal
-          closeButtonText="Annuleer"
+          closeButtonText="Nee"
+          confirmText="Ja"
```
Comment (robinpiets) [inline apps/client/src/components/Location/ChangeAddressModal.tsx:65]: As discussed...

**Review by robinpiets — CHANGES_REQUESTED** [review]
See comments.

File: apps/client/src/components/Nav.tsx [hunk 5]
```diff
@@ -33,7 +33,7 @@ const Nav: React.FC<NavProps> = ({
         <Button
           data-testid={NEXT_BUTTON}
           onClick={onGoToNext}
-          style={{ marginRight: formEnds ? 10 : 25 }}
+          style={{ marginRight: formEnds ? 10 : 25, textAlign: "center" }}
```
Comment (robinpiets) [inline apps/client/src/components/Nav.tsx]: Maybe better to import this as `NextButton` from `NavStyle`?

File: apps/client/src/components/AddressLine.tsx [hunk 6]
```diff
@@ -14,9 +15,11 @@ export default ({
   streetName,
 }: AddressLineProps) => (
   <>
-    {streetName} {houseNumberFull}
-    <div>
+    <Paragraph gutterBottom={0}>
+      {streetName} {houseNumberFull}
+    </Paragraph>
+    <Paragraph gutterBottom={16}>
```
Comment (afjlambert) [inline apps/client/src/components/AddressLine.tsx]: Somehow this component morphed from an addresline into AddresLines (plural). Can you update the filename to [reflect these changes](https://github.com/Amsterdam/vergunningcheck/pull/556/files)?
Also please add tests for `{postalCode} {residence}` if you have the time.

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/src/components/AddressLine.tsx (hunk 6 above)
Comment (svenjens) [reply apps/client/src/components/AddressLine.tsx]: There are already test for postalCode and Residence

**Review by robinpiets — APPROVED** [review]

**Pull request merged**